pyproject.toml

```diff
@@ -37,11 +37,13 @@ classifiers = [
 requires = [
     "alembic==1.5.4",
     "broadcaster[redis]==0.2.0",
+    "celery",
+    "celery[redis]",
     "click==8.0.3",
```

Comment:
Celery should be in a separate dependency chain for when the builtin executor is used

Reply:
I'll fix that before we merge this branch to main. For the moment we are testing with this branch.

Reply:
This setup is described in https://git.ia.surfsara.nl/netdev/automation/projects/orchestrator/-/issues/1572